SimGeneral/PreMixingModule/python/mixOne_premix_on_sim_cfi.py

```diff
@@ -314,3 +313,11 @@
         ),
     )
 )
+
+# Disable GE0 (as introduced in #31309)
+from Configuration.Eras.Modifier_phase2_GE0_cff import phase2_GE0
+phase2_GE0.toModify(mixData,
+    workers = dict(
+        me0 = None,
```

Comment:
Just to note that this could also be expressed on line 306 as
```py
(phase2_muon & ~phase2_GE0).toModify(...)
```

Reply:
Thanks @makortel 
Code is updated.